multimodal/src/autogluon/multimodal/models/utils.py

```diff
@@ -543,3 +543,34 @@ def get_hf_config_and_model(checkpoint_name: str, pretrained: Optional[bool] = T
         model = AutoModel.from_config(config)
 
     return config, model
+
+
+def get_mmocr_models(checkpoint_name: str):
```

Comment:
docstring

Reply:
Sorry,  I don’t catch that. How do I modify it ?

Reply:
Docstring means to add documentation to this function.

Reply:
You may refer to other functions. We use the numpy-style docstring to describe what the function does and what are the arguments.

Reply:
Thank you.